lib/src/model/documentation_comment.dart

```diff
@@ -676,6 +682,8 @@ mixin DocumentationComment
     for (var element in firstOfPair) {
       final result = element.group(2)!.trim();
       if (result.isEmpty) {
+        stderr.writeln("The warning 'missingCodeBlockLanguage' is deprecated. "
```

Comment:
Hmm, but this will only alert a developer to the deprecation if a new `missingCodeBlockLanguage` violation is found.

I think we need to alert (print to stderr) any time `dart doc` is invoked, with the missingCodeBlock warning enabled. Then for a codebase like Flutter (though we're holding their hand) will be notified via their CI that they are enabling a deprecated option.

Reply:
Oh you're right. Let me redo this. One sec.